src/test/kotlin/no/nav/familie/ef/sak/api/gui/VedtakControllerTest.kt

```diff
@@ -251,6 +270,68 @@ internal class VedtakControllerTest : OppslagSpringRunnerTest() {
         validerBehandlingIverksetter()
     }
 
+    @Nested
+    inner class AngreSendTilBeslutter {
+        @BeforeEach
+        fun setUp() {
+            mockBrukerContext(SAKSBEHANDLER.name)
+        }
+
+        @AfterEach
+        fun tearDown() {
+            clearBrukerContext()
+        }
+
+        @Test
+        internal fun `skal feile hvis en annen saksbehandler prøver å angre send til beslutter`() {
+            opprettBehandling(steg = StegType.BESLUTTE_VEDTAK, status = BehandlingStatus.FATTER_VEDTAK)
+            angreSendTilBeslutter(BESLUTTER, responseBadRequest())
+        }
+
+        @Test
+        internal fun `skal feile hvis vedtak ikke er i steg BESLUTTE_VEDTAK`() {
+            opprettBehandling(steg = StegType.SEND_TIL_BESLUTTER, status = BehandlingStatus.FATTER_VEDTAK)
+            angreSendTilBeslutter(SAKSBEHANDLER, responseBadRequest())
+        }
+
+        @Test
+        internal fun `skal feile hvis vedtak ikke har status FATTER_VEDTAK`() {
+            opprettBehandling(steg = StegType.BESLUTTE_VEDTAK, status = BehandlingStatus.UTREDES)
+            angreSendTilBeslutter(SAKSBEHANDLER, responseBadRequest())
+        }
+
+        @Test
+        internal fun `skal feile hvis oppgave er plukket av noen andre`() {
+            opprettBehandling(steg = StegType.BESLUTTE_VEDTAK, status = BehandlingStatus.FATTER_VEDTAK)
+            opprettOppgave(oppgaveType = Oppgavetype.GodkjenneVedtak, sakshandler = BESLUTTER)
+            angreSendTilBeslutter(SAKSBEHANDLER, responseBadRequest())
+        }
+
+        @Test
+        internal fun `skal kunne angre send til beslutter`() {
+            val behandlingId = opprettBehandling(steg = StegType.BESLUTTE_VEDTAK, status = BehandlingStatus.FATTER_VEDTAK)
```

Comment:
Skulle vi hatt en "enkel" test for `angreSendTilBeslutter` hvis oppgaven er plukket av saksbehandler?

```
            opprettBehandling(steg = StegType.BESLUTTE_VEDTAK, status = BehandlingStatus.FATTER_VEDTAK)
            opprettOppgave(oppgaveType = Oppgavetype.GodkjenneVedtak, sakshandler = SAKSBEHANDLER)
            angreSendTilBeslutter(SAKSBEHANDLER, responseOK())
```

Reply:
✅